oshmem/mca/spml/ucx/spml_ucx.c

```diff
@@ -294,6 +312,8 @@ int mca_spml_ucx_add_procs(ompi_proc_t** procs, size_t nprocs)
              free(mca_spml_ucx.remote_addrs_tbl[i]);
          }
     }
+    if (mca_spml_ucx_ctx_default.put_op_mask)
```

Comment:
we can call free without checking for NULL

Reply:
refactored